Sources/Testing/Testing.docc/AssociatingBugs.md

```diff
@@ -75,16 +75,16 @@ The testing library defines several kinds of common bug/test relationship:
 
 | Relationship | Use when… |
 |-|-|
-| `.uncoveredBug` | … a test run failed, uncovering the bug in question. |
-| `.reproducesBug` | … a bug was previously filed and the test was written to demonstrate it. |
-| `.verifiesFix` | … a bug has been fixed and the test shows that it no longer reproduces. |
-| `.failingBecauseOfBug` | … a test was previously passing, but now an unrelated bug is causing it to fail. |
-| `.unspecified` | … no other case accurately describes the relationship. |
+| `.uncoveredBug` | A test run failed, uncovering the bug in question. |
```

Comment:
The header still has an ellipsis. I'm not sure removing it from the rows results in the correct style?

Reply:
In this case, the ellipsis in the column header is OK, but with the cell contents, we don't need to follow the ellipsis pattern. This is mainly a table/list style thing.

Reply:
Alrighty!